WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -663,10 +663,10 @@ private Script GetCoordinatorScriptPreventReuse(Round round)
 	private void CoinVerifier_CoinBlacklisted(object? _, Coin coin)
 	{
 		// For logging reason Prison needs the roundId.
-		var roundState = RoundStates.FirstOrDefault(rs => rs.CoinjoinState.Inputs.Any(input => input.Outpoint == coin.Outpoint));
+		var round = Rounds.FirstOrDefault(round => round.Alices.Select(alice => alice.Coin.Outpoint).Contains(coin.Outpoint));
```

Comment:
My only question: Is it OK to iterate through the `Rounds` property without getting the `AsyncLock`?